cfe_internal/enterprise/templates/httpd.conf.mustache

```diff
@@ -222,7 +222,7 @@ LogLevel warn
     base-uri 'self'; \
     form-action 'self'; \
     media-src 'self'; \
-    worker-src 'self';"
+    worker-src 'self' blob:"
```

Comment:
@nickanderson asked me why no `;` at the end and I knew this worked but was curious.

https://www.w3.org/TR/CSP/

> A serialized CSP is an [ASCII string](https://infra.spec.whatwg.org/#ascii-string) consisting of a semicolon-delimited series of [serialized directives](https://www.w3.org/TR/CSP/#serialized-directive), adhering to the following ABNF grammar [[RFC5234]](https://www.w3.org/TR/CSP/#biblio-rfc5234):
serialized-policy =
    serialized-directive *( optional-ascii-whitespace ";" [ optional-ascii-whitespace serialized-directive ] )